doc/API/centreon-api-v2.yaml

```diff
@@ -4224,6 +4223,19 @@ components:
           type: string
           description: "Name of the group to which belongs the resource"
           example: "ESX-France|Cpus"
+    Monitoring.Notes:
+      type: object
+      properties:
+        notes_url:
```

Comment:
I think `notes` property should use following format : 
```
  notes: {
    label: 'my label',
    url: 'my url'
  }
```
cause currently `notes.notes` and `notes.notes_url` are a bit redondent

Reply:
alright 👍